### PR TITLE
Feature/heritage 231

### PR DESCRIPTION
**Pull request by JohnHeeryTNA**
Ticket URL: https://national-archives.atlassian.net/browse/HERITAGE-231

## About these changes

Timeline / Histogram views being added in

## How to check these changes

Pull branch and run locally, check the timeline views
http://127.0.0.1:8000/search/catalogue/?group=community&vis_view=timeline&timeline_type=century

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes
- [x] Ensured that PR includes only commits relevant to the ticket
- [x] Waited for all CI jobs to pass before requesting a review
- [x] Added/updated tests and documentation where relevant

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

### CODE REVIEW

File: templates/search/blocks/visualisation_menu.html [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {% load search_tags %}
 
 <!--visualisation secondary navigation-->
-<div class="tna-container">
+<div class="tna-container">
```
Comment (TNA-Allan) [inline templates/search/blocks/visualisation_menu.html:4]:
```suggestion
<div class="tna-container">
```
Blank space introduced. As there is no code change, space can be removed and restored.

File: scripts/src/modules/map/data/timeline.json
Comment (TNA-Allan) [inline scripts/src/modules/map/data/timeline.json:1]: There are two of these files. Can we do with one?

**Review by TNA-Allan — CHANGES_REQUESTED** [review]
Got some errors on page-inspect using Chrome.

1.  http://127.0.0.1:8000/search/catalogue/?group=community&vis_view=timeline&timeline_type=century

Invalid scale configuration for scale: xAxes chart.js@3.9.1:13 

2. http://127.0.0.1:8000/search/catalogue/?vis_view=timeline&timeline_type=decade&creation_date_from=1800#decade-counts-chart

Refused to apply style from 'http://127.0.0.1:8000/search/catalogue/css/styles.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.

3.  http://127.0.0.1:8000/search/catalogue/?vis_view=timeline&timeline_type=year&creation_date_from=1860#decade-counts-chart

Refused to apply style from 'http://127.0.0.1:8000/search/catalogue/css/styles.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.

**Comment by JohnHeeryTNA** [comment]
> Got some errors on page-inspect using Chrome.
> 
> 1. http://127.0.0.1:8000/search/catalogue/?group=community&vis_view=timeline&timeline_type=century
> 
> Invalid scale configuration for scale: xAxes chart.js@3.9.1:13
> 
**This was due to the version of chartJS we are using, I've adapted the code so the error does not occur.**


> 2. http://127.0.0.1:8000/search/catalogue/?vis_view=timeline&timeline_type=decade&creation_date_from=1800#decade-counts-chart
> 
> Refused to apply style from 'http://127.0.0.1:8000/search/catalogue/css/styles.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
> 
> 3. http://127.0.0.1:8000/search/catalogue/?vis_view=timeline&timeline_type=year&creation_date_from=1860#decade-counts-chart
> 
> Refused to apply style from 'http://127.0.0.1:8000/search/catalogue/css/styles.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.

I've removed these style rules as the css is now handled by the sass stack

File: scripts/src/modules/map/data/timeline.json
Comment (JohnHeeryTNA) [reply scripts/src/modules/map/data/timeline.json:1]: I've removed the duplicate

**Review by TNA-Allan — APPROVED** [review]
Looks good to me. Thanks @JohnHeeryTNA

**Pull request merged**